api/dns.go

```diff
@@ -21,6 +21,6 @@ func (d *DNSHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	logger := loggerFromContext(r.Context())
 	log(logger, "Performing DNS")
 
-	Write(w, http.StatusOK, `{"domains": ["test.example.test", "test2.example.test"]}`)
+	Write(w, http.StatusOK, `{"domains": ["mystack.com"]}`)
```

Comment:
I must have misunderstood your idea, then. I did this so that the CLI gets every request that contains 'mystack.com' as substring and routes them using our DNS server. If it is an inexistent route, 404 is returned. Otherwise the request goes on.